scripts/protocgen.sh

```diff
@@ -29,5 +29,5 @@ Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
   $(find "${dir}" -maxdepth 1 -name '*.proto')
 done
 
-cp -r ./github.com/osmosis-labs/osmosis/* ./
+cp -r ./github.com/osmosis-labs/osmosis/v*/x/* x/
```

Comment:
Oh cool, did you test make proto-gen on this branch and it works?

Reply:
Yep! With this included, proto doesn't get confused where it should generate the proto files even with the go semantic versioning included!

Reply:
yup